include/assimp/material.h

```diff
@@ -1000,6 +1000,7 @@ extern "C" {
 // Specular Color.
 // Note: Metallic/Roughness may also have a Specular Color
 // AI_MATKEY_COLOR_SPECULAR
+#define AI_MATKEY_USE_GLTF_PBR_SPECULAR_GLOSSINESS "$mat.useGltfPbrSpecularGlossiness", 0, 0
```

Comment:
Sorry, I didn't explain properly.
This should not be an AI_MATKEY_, it should be an exporter property.

Exporter properties are set by the application to customise the behaviour of the exporter, they're not part of the model.

`glTF2Exporter(const char *filename, IOSystem *pIOSystem, const aiScene *pScene,
            const ExportProperties *pProperties, bool binary);`

See:
https://github.com/assimp/assimp/blob/cfed74516b46a7c2bdf19c1643c448363bd90ad7/include/assimp/Exporter.hpp#L372

Reply:
> Sorry, I didn't explain properly. This should not be an AI_MATKEY_, it should be an exporter property.
> 
> Exporter properties are set by the application to customise the behaviour of the exporter, they're not part of the model.

This creates an issue, if an asset is imported that uses the pbrSpecGlos extension there will be no way to guarantee that the exporter will use it since the importer can't use the SetPropertyBool of the exporter. 
With my implementation the imported material will havethe defined flag so that when the same model is exported that material will be unchanged and will also have the pbrSpecGlos extension.
Thoughts?